### PR TITLE
doc: typo fixes

### PR DESCRIPTION
**Pull request by ghost**
## Description

Fix some small typos.

## Related Issues

N/A

### Check List

- [X] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [X] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [X] I have run `npx turbo format` to ensure the code follows the style guide.
- [ ] I have run `npx turbo test` to check if all tests are passing, and/or `npx turbo test:snapshot` to update snapshots if I created and/or updated React Components.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Review by AugustinMauroy — APPROVED** [review]
LGTM!

**Review by mikeesto — APPROVED** [review]

**Review by shanpriyan — APPROVED** [review]
LGTM!

**Review by ovflowd — APPROVED** [review]

**Comment by ovflowd** [comment]
cc @nodejs/website for future references, such small changes can be immediately merged once you get an approval (not even fast-tracking is needed) as per our contributing guidelines :)

**Pull request merged**
